gammapy/modeling/parameter.py

```diff
@@ -357,26 +357,27 @@ def _step(self):
     # TODO: possibly allow to set this independently
     @property
     def conf_min(self):
-        """Confidence minimum value as a `float`.
-
-        Return parameter minimum if defined, otherwise return the scan_min.
-        """
+        """Confidence minimum value as a `float`."""
```

Comment:
```suggestion
        """Confidence minimum value as a `float`.
        Return parameter minimum if defined, otherwise  a default is estimated from value and error.
        """
```